### ISSUE TITLE
Two-stage Long Videos Generation not implement yet?

### ISSUE THREAD

**Issue by janoschsimon** [HVision-NKU/StoryDiffusion#22]
Hey first of all congrats on that amazing Project :) i got the gradio_app_sdxl_specific_id.py running and it produce some nice comic strips. but where is the video stored? or is that feature not in the gradio demo and comes later?

thx janosch

**Comment by zhoudaquan** [comment]
> Hey first of all congrats on that amazing Project :) i got the gradio_app_sdxl_specific_id.py running and it produce some nice comic strips. but where is the video stored? or is that feature not in the gradio demo and comes later?
> 
> thx janosch

Hi，

Thanks for your interest. The video-related code is still under legal inspection from the company side.

Regards,
DQ

**Comment by janoschsimon** [comment]
makes total sense i re-read all the posts and was like damnnnnn what am i missing :) but the comic strip works very well! keep up the great work 
Janosch

**Comment by dyter07** [comment]
Yeah, I was just looking for the video functions. Hope to see a release shortly

**Comment by Speedway1** [comment]
Comic strip technology is very good though, shows how to get character consistency.

**Comment by 3blackbar** [comment]
> Comic strip technology is very good though, shows how to get character consistency.

any result examples ?

**Comment by Speedway1** [comment]
> > Comic strip technology is very good though, shows how to get character consistency.
> 
> any result examples ?

This was meant to be a spy thing with a female spy sharing results but didn't quite work out. 

![image](https://github.com/HVision-NKU/StoryDiffusion/assets/100301611/42f398f6-b038-4765-8721-d7d5cfa4ad0f)
